compiler/src/Language/Bond/Codegen/Cpp/ApplyOverloads.hs

```diff
@@ -22,20 +23,22 @@ data Protocol =
 
 
 -- Apply overloads
-applyOverloads :: [Protocol] -> Text -> Text -> Declaration -> Text
-applyOverloads protocols attr body Struct {..} | null declParams = [lt|
+applyOverloads :: [Protocol] -> MappingContext -> Text -> Text -> Declaration -> Text
+applyOverloads protocols cpp attr body s@Struct {..} | null declParams = [lt|
     //
     // Overloads of Apply function with common transforms for #{declName}.
     // These overloads will be selected using argument dependent lookup
     // before bond::Apply function templates.
     //
-    #{attr}bool Apply(const bond::To<#{declName}>& transform,
-               const bond::bonded<#{declName}>& value)#{body}
+    #{attr}bool Apply(const bond::To< #{qualifiedName}>& transform,
```

Comment:
Without those spaces, the build breaks for me.

    [ 50%] Building CXX object cpp/CMakeFiles/bond_apply.dir/bond/core/bond_apply.cpp.o
    In file included from /Users/ted/code/bond/build/cpp/bond/core/bond_apply.cpp:13:
    /Users/ted/code/bond/build/cpp/bond/core/bond_apply.h:28:30: error: found '<::' after a template name which forms the digraph '<:' (aka '[') and a ':', did you mean '< ::'?
        bool Apply(const bond::To<::bond::SerializableExceptionBase>& transform,
                                 ^~~
                                 < ::